assets/css/styles.css

```diff
@@ -235,23 +310,43 @@ footer {
   z-index: 101;
 }
 
+
 /*rotate element animation using only CSS*/
 
-.rotate-in-x {
-  animation: rotate-in-x linear;
+.rotate-in {
+  animation: rotate-in linear;
   animation-timeline: view();
-  animation-range: entry -10% entry 90%;
+  animation-range: entry -10% entry 70%;
 }
 
-@keyframes rotate-in-x {
+@keyframes rotate-in {
   from {
     opacity: 0.35;
-    transform: rotateX(90deg) rotateY(45deg);
-    z-index: 0;
+    transform: rotateX(45deg) rotateY(45deg);
   }
   to {
     opacity: 1;
     transform: rotateX(0deg) rotateY(0deg);
-    z-index: 90;
   }
-}
\ No newline at end of file
+}
+
+
+/*animated cursor*/
+
+ .masked {
+  z-index: -100;
+  max-width: 150%;
+  opacity: 50%;
+  object-fit: cover;
+  -webkit-mask-image: url(../images/gradient-image.svg);
+  -webkit-mask-repeat: no-repeat;
+  -webkit-mask-size: 0em;
+  transition: 120ms linear;
+}
+
+@media only screen and (max-aspect-ratio: 1/1) {
+  .masked {
+    visibility: hidden;
+  }
+}
+

```

Comment:
I would recommend ordering rulesets in CSS by order of specificity. More general styles should be at the top (mainly element selectors like `body`, `html`, `p`) and more specific styles should come after.

As an example, styling text elements would probably look like this:
```
h1, h2, h3, h4, h5, h6 {
  font-family: Times;
  color: blue;
}

h1 {
  color: black;
  font-size: 2em;
}
```

Reply:
Very good point. I like this idea. I will address this as a separate issue (see issue #12 ).
Thanks!